index.html

```diff
@@ -1,162 +1,112 @@
-            <button class="task-list-item__button task-list-item__button--is-done">&check;</button> 
-            
-          <label class="task-list-item__content">
-              Meet with friends
-          </label>
-          <button class="task-list-item__button task-list-item__button--delete">&#10060</button>
-        </li> 
-        <li class="task-list__item">
-          <button class="task-list-item__button task-list-item__button--is-done">&check;</button> 
-          
-        <label class="task-list-item__content">
-            Meet with friends
-        </label>
-        <button class="task-list-item__button task-list-item__button--delete">&#10060</button>
-      </li> 
-      <li class="task-list__item">
-        <button class="task-list-item__button task-list-item__button--is-done">&check;</button> 
-        
-      <label class="task-list-item__content">
-          Meet with friends
-      </label>
-      <button class="task-list-item__button task-list-item__button--delete">&#10060</button>
-    </li> 
-    
-        </ul>
-      </form>
+				<ul class="task-list">
+					<li class="task-list__item">
+						<button class="task-list-item__button task-list-item__button--is-done" type="button">
+							&#10003;
+						</button>
 
-      <div class="task-form__wrapper">
-      <form class="task-form task-form__done">
-        <label class="task-form__title">Done</label>
+						<span class="task-list-item__content">
+							Go to the cinema
+						</span>
 
-      </form>
+						<time datetime="2019-04-20" class="task-list-item__deadline">
+							2019-04-20
+						</time>
+						<button class="task-list-item__button task-list-item__button--delete" type="button">
+							&#10060;
+						</button>
+					</li>
+					<li class="task-list__item">
+						<button class="task-list-item__button task-list-item__button--is-done" type="button">
+							&#10003;
+						</button>
 
-      <form class="task-form task-form__deleted">
-        <label class="task-form__title">Deleted</label>
+						<span class="task-list-item__content">
+							Do homework
+						</span>
 
-      </form>
-    </div>
-    </div>
-  </body>
+						<time datetime="2019-04-20" class="task-list-item__deadline">
+							2019-04-20
+						</time>
+						<button class="task-list-item__button task-list-item__button--delete" type="button">
+							&#10060;
+						</button>
+					</li>
+
+					<li class="task-list__item">
+						<button class="task-list-item__button task-list-item__button--is-done" type="button">
+							&#10003;
+						</button>
+						<span class="task-list-item__content">
+							Meet with friends
+						</span>
+						<time datetime="2019-04-20" class="task-list-item__deadline">
+							2019-04-20
+						</time>
+						<button
+							type="submit"
+							class="task-list-item__button task-list-item__button--delete"
+							type="button"
+						>
+							&#10060;
+						</button>
+					</li>
+				</ul>
+			</div>
+
+			<div class="task-form__wrapper">
```

Comment:
`task-form` prefix doesn't match up with the actual content of this block. There're two lists, not forms.